src/App/Header/Search.svelte

```diff
@@ -107,11 +111,6 @@
   .search {
     transition: all 0.2s;
   }
-  @media (max-width: 720px) {
-    .search {
-      display: none;
-    }
-  }
 </style>
 
 <div class="search" style:width={`${searchBarWidth}rem`}>
```

Comment:
wrt to my first comment here we could do something like this
```suggestion
<div class="search" class:collapsed={isCollapsed} class:expanded={!isCollapsed}>
```